src/lib/github.ts

```diff
@@ -0,0 +1,44 @@
+import * as core from "@actions/core";
+import * as github from "@actions/github";
+import {} from "@actions/github/lib/github";
+import { GitHub } from "@actions/github/lib/utils";
+
+let client: InstanceType<typeof GitHub>;
+function getClient(): InstanceType<typeof GitHub> {
+  if (client) {
+    return client;
+  } else {
+    const GITHUB_TOKEN = core.getInput("github-token");
+    return github.getOctokit(GITHUB_TOKEN);
+  }
+}
+
+type ActionPayload = {
+  repo: {
+    owner: string;
+    repo: string;
+  };
+  prNumber: number;
+};
+
+async function comment({ repo, prNumber }: ActionPayload): Promise<void> {
+  const octokit = getClient();
+
+  await octokit.rest.issues.createComment({
+    ...repo,
+    issue_number: prNumber,
+    body: "@dependabot squash and merge",
+  });
+}
+
+async function review({ repo, prNumber }: ActionPayload): Promise<void> {
```

Comment:
```suggestion
async function approve({ repo, prNumber }: ActionPayload): Promise<void> {
```
if I read your code right 😁

Reply:
Yep, you're right, so now we have:
- review -> approve
- comment -> squashAndMerge

This might change again if we need more of them